Tests/BeaconChainTests/StateTransitionTests.swift

```diff
@@ -51,4 +51,57 @@ final class StateTransitionTests: XCTestCase {
             )
         }
     }
+
+    func testExpectedFFGTarget() {
+        // @todo check these numbers
+        let tests = [
```

Comment:
https://github.com/prysmaticlabs/prysm/blob/master/beacon-chain/core/balances/rewards_penalties_test.go#L131-L133 The output numbers don
t seem to match prysmatics expectations, this could be because they are still using `MAX_DEPOSIT_IN_GWEI`?